5.5/5.5.9/index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 5.5.9
+Напишите функцию shuffle(array), которая перемешивает (переупорядочивает случайным образом) элементы массива.
+Многократные прогоны через shuffle могут привести к разным последовательностям элементов. Например:
+let arr = [1, 2, 3];
+
+shuffle(arr);
+// arr = [3, 2, 1]
+
+shuffle(arr);
+// arr = [2, 1, 3]
+
+shuffle(arr);
+// arr = [3, 1, 2]
+// ...
+
+Все последовательности элементов должны иметь одинаковую вероятность. Например, [1,2,3] может
+быть переупорядочено как [1,2,3] или [1,3,2], или [3,1,2] и т.д., с равной вероятностью каждого случая.
+<script>
+	function compare(a, b) {
+		if (a > b) return Math.floor((Math.random() * 2) - 1);
+	}
+
+	function shuffle(array) {
+		return array.sort(compare);
+	}
+
+	let arr = [1, 2, 3];
+
+	shuffle(arr);
+	shuffle(arr);
+	shuffle(arr);
```

Comment:
the 'shuffle' function doesn't work, we always have the same result when we run it a few times:
![image](https://github.com/tacya9/JS-1-5/assets/115877101/a4bff9bd-2d79-401f-a5d0-836c4c4453f9)